catalogue_pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/model/MergedWork.scala

```diff
@@ -0,0 +1,8 @@
+package uk.ac.wellcome.platform.merger.model
+import uk.ac.wellcome.models.work.internal.{
+  UnidentifiedRedirectedWork,
+  UnidentifiedWork
+}
+
+case class MergedWork(work: UnidentifiedWork,
```

Comment:
tyyyyyyyyypes

Reply:
ah this shouldn't have been committed. It was an attempt at going down a typed route, but it turned out to be a pain

Reply:
No, sorry I'm getting confused. This is actually fine. What was your comment about?

Reply:
That was meant in an enthusiastic way. A typed structure over a list == win! :-)